src/common/classes/ClumpletReader.cpp

```diff
@@ -311,6 +308,9 @@ ClumpletReader::ClumpletType ClumpletReader::getClumpletType(UCHAR tag) const
 		case isc_spb_auth_plugin_name:
 		case isc_spb_auth_plugin_list:
 			return Wide;
+		default:
```

Comment:
@AlexPeshkoff  I'm thinking this entire switch is unnecessary, as the other branches returning `Wide` implies kind should be `WideTagged`, not `SpbStart`, right?

Reply:
Also do not understand what do this options do here - they all are from SpbAttach. Kill it please.